src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -281,13 +294,33 @@ void compare(const ov::Tensor& expected,
     if (abs_threshold == std::numeric_limits<double>::max() && rel_threshold == std::numeric_limits<double>::max()) {
         if (sizeof(ExpectedT) == 1 || sizeof(ActualT) == 1) {
             abs_threshold = 1.;
+            rel_threshold = 1.;
```

Comment:
@sbalandi Please check the changes on all platforms and skip tests per platform (in case some test is failed only on one platform)